posthog/api/capture.py

```diff
@@ -296,7 +296,7 @@ def get_event(request):
         ingestion_context = None
         send_events_to_dead_letter_queue = False
 
-        if token in settings.LIGHTWEIGHT_CAPTURE_ENDPOINT_ENABLED_TOKENS:
+        if settings.LIGHTWEIGHT_CAPTURE_ENDPOINT_ALL or token in settings.LIGHTWEIGHT_CAPTURE_ENDPOINT_ENABLED_TOKENS:
```

Comment:
I'd even just do it here since it's only used in a single place anyway
```suggestion
        if settings.LIGHTWEIGHT_CAPTURE_ENDPOINT_ALL == '*' or token in settings.LIGHTWEIGHT_CAPTURE_ENDPOINT_ENABLED_TOKENS:
```

Reply:
This is executed for each event, that's why I deemed it better to check the condition once at startup.